saleor/static/js/components/language-picker.js

```diff
@@ -0,0 +1,7 @@
+$('.language-pick').on('click', (e) => {
+  const $target = $(e.currentTarget);
+  const $langCode = $target.attr('js-lang');
+  const $input = $(`<input name="language" type="hidden" value="${$langCode}">`);
+  $('#language-picker').append($input);
+  $('#language-picker').submit();
```

Comment:
I think it would be better to put the result of `$('#language-picker')` into a `const`.